extensions/HyperVExtension/src/HyperVExtension/Models/VMGalleryCreationAdaptiveCardSession.cs

```diff
@@ -213,6 +213,7 @@ private async Task<ProviderOperationResult> GetForReviewFormAdaptiveCardAsync(st
 
             var galleryImage = _vMGalleryImageList.Images[inputForGalleryOperation.SelectedImageListIndex];
             var newEnvironmentNameLabel = _stringResource.GetLocalized("NameLabelForNewVirtualMachine", ":");
+            var microsoftLogoAltText = _stringResource.GetLocalized("MicrosoftLogoAltText");
```

Comment:
>MicrosoftLogoAltText [](http://example.com/codeflow?start=69&length=20)

I'm not seeing this string in the resw

Reply:
Ah this is just no longer used and should be deleted

Reply:
I forgot to remove it.